package.json

```diff
@@ -29,6 +29,7 @@
     "lint": "tslint -c tslint.json -p tsconfig.json --fix"
   },
   "jest": {
+    "testURL": "http://localhost",
```

Comment:
See https://github.com/facebook/jest/issues/6766#issuecomment-408344243

I wasn't able to run tests locally with the newest version of Jest without this.